### PR TITLE
Add archway-bitcanna IBC

### PR DESCRIPTION
**Pull request by sigv**
`IBC Channel Open Confirm` on [BitCanna Tx 8CAC6901342ACD758AAA1BBEB116FEC9F7DBB199168217C8AB244D7836D30C64](https://explorer.bitcanna.io/transactions/8CAC6901342ACD758AAA1BBEB116FEC9F7DBB199168217C8AB244D7836D30C64).

### CODE REVIEW

File: _IBC/archway-bitcanna.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
+{
+  "$schema": "../ibc_data.schema.json",
+  "chain_1": {
+    "chain_name": "archway",
```
Comment (shahbazn) [inline _IBC/archway-bitcanna.json:4]: archway-1

File: _IBC/archway-bitcanna.json [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    "client_id": "07-tendermint-35",
```
Comment (shahbazn) [inline _IBC/archway-bitcanna.json:5]: client_id: 07-tendermint-35 trusting period is not 2/3 of the unbodning period

>         trusting_period: 604800s,
>         unbonding_period: 1209600s,

Comment (shahbazn) [reply _IBC/archway-bitcanna.json:5]: trusting period must be: 806400s to comply with requirements [here](https://github.com/archway-network/networks#requirements-for-the-ibc-channel)

Comment (sigv) [reply _IBC/archway-bitcanna.json:5]: > client_id: 07-tendermint-35 trusting period is not 2/3 of the unbodning period

I am trying to figure out how these values ended up here.

> trusting period must be: 806400s to comply with requirements [here](https://github.com/archway-network/networks#requirements-for-the-ibc-channel)

As per the referenced link, "trusting period must be 1209600s".

---

Why this is confusing to me is Hermes should pick up unbonding period auto-magically and the trusting period was specified by me as `14days` which is the expected `1209600s`, so the client should have been generated as:
> ```text
> trusting_period: 1209600s,
> unbonding_period: 1814400s,
> ```

I will try to figure out why it ended up the way it did, and update this PR with new client/connection/channel.

Comment (sigv) [reply _IBC/archway-bitcanna.json:5]: Got it - it's a misconfiguration for our BitCanna side.

**Review by shahbazn — APPROVED** [review]
looks good thanks for the fixes. lgtm

**Comment by sigv** [comment]
Thank you too, for double-checking!

**Pull request merged**